src/som/interpreter/actors/Actor.java

```diff
@@ -185,7 +191,7 @@ protected void appendToMailbox(final EventualMessage msg) {
   }
 
   protected static void handleBreakPoints(final EventualMessage msg, final WebDebugger dbg) {
-    if (VmSettings.TRUFFLE_DEBUGGER_ENABLED && msg.isBreakpoint()) {
+    if (VmSettings.TRUFFLE_DEBUGGER_ENABLED && msg.isMessageReceiverBreakpointSet()) {
```

Comment:
*sigh* I think I have been asking for that change already a long time ago, no?
It would be really great if you could just split your work into smaller increments and we integrate stuff more often.
Avoids double work for both of us.